APPS.md

```diff
@@ -1,48 +1,86 @@
-| [tenere](https://github.com/pythops/tenere)                          | TUI interface for LLMs written in Rust                                                                    | [Badr Badri](https://github.com/pythops)                 |
-| [termchat](https://github.com/lemunozm/termchat)                     | Terminal chat through the LAN with video streaming and file transfer                                      | [Luis Enrique Muñoz Martín](https://github.com/lemunozm) |
-| [termscp](https://github.com/veeso/termscp)                          | A feature rich terminal UI file transfer and explorer with support for SCP/SFTP/FTP/S3                    | [Christian Visintin](https://github.com/veeso)           |
-| [tick-rs](https://github.com/tarkah/tickrs)                          | Realtime ticker data in your terminal                                                                     | [Cory Forsstrom](https://github.com/tarkah)              |
-| [trippy](https://github.com/fujiapple852/trippy)                     | A network diagnostic tool (combines traceroute & ping, like mtr)                                          | [fujiapple852](https://github.com/fujiapple852)          |
-| [tsuchita](https://github.com/kamiyaa/tsuchita)                      | Client-server notification center for dbus desktop notifications                                          | [Jeff Zhao](https://github.com/kamiyaa)                  |
-| [tuinance](https://github.com/landchad/tuinance)                     | Display financial data on the terminal                                                                    | [bloatoo](https://github.com/bloatoo)                    |
-| [twitch-tui](https://github.com/Xithrius/twitch-tui)                 | Twitch chat in the terminal.                                                                              | [Xithrius](https://github.com/Xithrius)                  |
-| [vector](https://vector.dev)                                         | A lightweight, ultra-fast tool for building observability pipelines                                       | [vectordotdev](https://github.com/vectordotdev)          |
-| [xplr](https://github.com/sayanarijit/xplr)                          | A hackable, minimal, fast TUI file explorer                                                               | [Arijit Basu](https://github.com/sayanarijit/xplr)       |
-| [ytop](https://github.com/cjbassi/ytop)                              | A TUI system monitor written in Rust (no longer maintained)                                               | [Caleb Bassi](https://github.com/cjbassi)                |
-| [zenith](https://github.com/bvaisvil/zenith)                         | Sort of like top or htop but with zoom-able charts, CPU, GPU, network, and disk usage                     | [Benjamin Vaisvil](https://github.com/bvaisvil)          |
+#### 💻 Development Tools:
```

Comment:
I suggest we drop the ":' at the end of the titles.

Reply:
done :+1: